### ISSUE TITLE
Review from Tatuya Jinmei (INT)

### ISSUE THREAD

**Issue by beurdouche** [mlswg/mls-architecture#177]
```Reviewer: Tatuya Jinmei
Review result: Ready with Nits

I am an assigned INT directorate reviewer for
draft-ietf-mls-architecture-10.txt. These comments were written primarily for
the benefit of the Internet Area Directors. Document editors and shepherd(s)
should treat these comments just like they would treat comments from any other
IETF contributors and resolve them along with any other Last Call comments that
have been received. For more details on the INT Directorate, see
https://datatracker.ietf.org/group/intdir/about/.

I've reviewed the draft in terms of issues or concerns for the Internet Area.
I've not seen any such issues. And, overall, the document is well written and
looks complete for publication.

The only possible nit I might note is that some technical terms such as
"Proposal" or "Commit" messages are used without a reference or definition,
which confused me while reviewing the draft. From a quick look at the protocol
draft, these may be defined in that draft, but in any case, it would be more
reader friendly if the architecture draft has a "terminology" section, listing
some of those terms with a brief description and or a proper reference.```

**Comment by rohan-wire** [comment]
NIT addressed in PR #182

**Comment by beurdouche** [comment]
Verified as fixed, Thank you for the Review !
